.gitignore

```diff
@@ -1,59 +1,149 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Do you really have to rewrite the whole file? Just add the Rules that are missing.